prisma/migrations/20240902061032_add_user_claims/migration.sql

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I ended up generating a new migration, is this fine or is there a way to merge this with the already present one?

Reply:
That's fine! There's no way to merge other than deleting existing migrations and restarting our migration history. I might do that so things are cleaner, as the migrations don't really matter at this stage because our production db hasn't had any writes made. Once we start taking new ratings, we'll only ever add migrations.

Reply:
I generated a new [baseline migration](https://www.prisma.io/docs/orm/prisma-migrate/getting-started#create-a-baseline-migration) to give us a clean migration history before we start accepting user data. For my future reference (and anyone curious :D) here are the steps I used (step 4 differs from the linked guide, as I ran into a bug where Prisma couldn't find my baseline migration at step 5):
1. Delete the **CONTENTS ONLY** of the `_prisma_migrations` table from your local db, if applicable.
2. Create a fresh export of your local db, if applicable.
3. Delete the contents of `prisma/migration`.
4. Run this command: `npx prisma migrate dev --create-only --name 0_init --skip-seed --skip-generate`.
5. Rename the generated migration folder from `YYYYMMDDhhmmss_0_init` to `0_init`.
6. Drop your local `modslist` database, recreate it, and import your fresh export.
7. Run this command: `npx prisma migrate resolve --applied 0_init`.
8. Commit the results.